cloudevents/extensions/bam.md

```diff
@@ -0,0 +1,127 @@
+# Business Activity Monitoring (BAM) Extension
+
+Business Activity Monitoring (BAM) was originally coined by analysts at
+Gartner, and refers to the aggregation, analysis, and presentation of
+real-time information about activities inside organizations, customers,
+and partners.
+
+The activity monitoring consists of a model of a business process,
+which can consists of multiple transactions (e.g. order, payment, invoice),
+and these transactions can have multiple steps. The technical processing
+represented by a transaction instance `bamtxid` is then correlated with
+the steps in those transactions of the business process.
+
+This extension defines attributes that can be included within a CloudEvent
+to describe the business activity that the event is associated with.
+
+Producers and consumers are free to define an out-of-band agreement on the
+semantic meaning, or valid values, for the attribute.
+
+## Notational Conventions
+
+As with the main [CloudEvents specification](../spec.md), the key words "MUST",
+"MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT",
+"RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as
+described in [RFC 2119](https://tools.ietf.org/html/rfc2119).
+
+However, the scope of these key words is limited to when this extension is
+used. For example, an attribute being marked as "REQUIRED" does not mean
+it needs to be in all CloudEvents, rather it needs to be included only when
+this extension is being used.
+
+## Attributes
+
+### bamtxid (Business Transaction ID)
```

Comment:
minor thing, but I wonder if the stuff in parens should be `BAM Transaction ID`? Then all of the letters in the attribute name are correlated with the descriptive name - as of now we're missing the words for "am"

Reply:
ping @katallaxie, any thoughts on ^^ ?

Reply:
@duglin I changed it to `BAM` as we are defining the `BAM` term in the header. I think then this is consistent.